README.md

```diff
@@ -31,69 +40,14 @@ Install dependencies with, for example:
 ```sh
 python3.10 -m pip install -r requirements.txt
 ```
-
-## Configuration
-
-### iRODS environment.json
-
-- Please create a directory/folder named `.irods` in your home directory/folder (`~/.irods/` in Linux shorthand).
```

Comment:
I assume this whole block has been relocated to elsewhere in the documentation.  I have not seen it in a new home in this PR.

Reply:
Yes, we have a real Documentation webpage. All markdown files for these pages are in the docs folder. I removed it here to not have to update always two sources of information. The README just contains the absolute necessary information for the user to decide whether the system requirements match his/her system.

Reply:
Oki doki!